### DISCUSSION TITLE
Part 1 HW2

### DISCUSSION THREAD

**Discussion by UniquelyAbbi** [cooperrc/computational-mechanics#46] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/106557922/172963185-88a49fe9-4c2f-480e-944a-c77b8de6a81f.png)
Hello, i need some feedback please, I tried doing pd.read_cvs and also pd.column_cvs.

**Comment by feliciones** [comment, marked as answer]
Instead of your transistor data line and using the _cvs, I would suggest using this line `transistor_data.columns` or `print("The columns for this data set are", transistor_data.columns)`

**Reply by UniquelyAbbi** [reply]
I understand your suggestion but it keeps saying i didnt define it.

**Reply by feliciones** [reply]
Try this which first defines the transistor data before calling upon it:

!head ../data/transistor_data.csv
transistor_data = pd.read_csv('../data/transistor_data.csv')
print("The headings of the columns for this data set are", transistor_data.columns)

**Reply by UniquelyAbbi** [reply]
ohhhh i was missing the data inside the (). Thank you

**Reply by UniquelyAbbi** [reply]
I think the problem is with the pd.read_cvs

**Reply by UniquelyAbbi** [reply]
AttributeError: module 'pandas' has no attribute 'read_cvs'

**Reply by feliciones** [reply]
@UniquelyAbbi Would you mind marking my reply as the answer so other students know that your question has been answered and can reference it? It makes it easier to sort when you are looking for help as I usually do lol.

**Reply by UniquelyAbbi** [reply]
@cooperrc  what does AttributeError: module 'pandas' has no attribute 'read_cvs' mean. is there a way to fix it?

**Reply by cooperrc** [reply]
> @cooperrc  what does AttributeError: module 'pandas' has no attribute 'read_cvs' mean. is there a way to fix it?

~`read_cvs`~ -> `read_csv`

Its a typo, you're reading the Comma Separated Value (CSV) file.
